test/studies/shootout/fasta/kbrady/fasta-enum.chpl

```diff
@@ -0,0 +1,161 @@
+/* The Computer Language Benchmarks Game
+ * http://benchmarksgame.alioth.debian.org/
+ *
+ * contributed by Kyle Brady
+ * modified from the Chapel version by Casey Battaglino
+ * edited by Preston Sahabu
```

Comment:
You can probably just add your name after the contributed line here and in the other version, I think these changes are significant enough to warrant it.